openwisp_notifications/templates/emails/batch_email.txt

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+[{{ site_name }}] {{ notifications_count }} {% translate "new notifications since" %} {{ start_time }}
+
+{% for notification in notifications %}
+- {{ notification.message }}
+  {% translate "URL" %}: {{ notification.url }}
+  {% translate "Timestamp" %}: {{ notification.timestamp|date:"F j, Y, g:i a" }}
+  {% if show_notification_description %}
+      {% translate "Description" %}: {{ notification.rendered_description }}
+  {% endif %}
+{% endfor %}
+
+{% if not show_notification_description  %}
+{% translate "View all Notifications" %}: {{ call_to_action_url }}
```

Comment:
In the test output shown by @pandafy the URL is not showing.

Reply:
```suggestion
{{ call_to_action_text }}: {{ call_to_action_url }}
```

Reply:
Updated!